app/index.js

```diff
@@ -12,6 +12,8 @@ process.on('SIGINT', async () => {
   process.exit(0)
 })
 
-module.exports = (async function startService () {
+const startService = async () => {
   await messageService.start()
-}())
+}
+
+module.exports = { startService }
```

Comment:
This will stop the service starting up.  The original notation is a way of handling top level awaits.  ie the async function is wrapped in a function and immediately executed.  

The change here although meeting the criteria of changing to an arrow function is only exporting the function and not executing it.  Therefore nothing executes it and the service immediately shuts down.

If you run the `./scripts/start` script you'll see the container immediately exits, rather than waiting for payments.

![image](https://user-images.githubusercontent.com/42574977/205121282-4bcbad90-62b8-4416-8659-c86a15f01c7a.png)

As a spoiler you can fix this by changing the original to this -> 

![image](https://user-images.githubusercontent.com/42574977/205121782-1e7d9f12-d445-4fcf-b2c5-82717cafb29c.png)